src/storage/src/monitor/relabeled_metric.rs

```diff
@@ -0,0 +1,93 @@
+// Copyright 2023 RisingWave Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use prometheus::core::{GenericCounter, GenericCounterVec};
+use prometheus::{Histogram, HistogramVec};
+
+pub const DEFAULT_STORAGE_METRIC_LEVEL: u8 = 4;
```

Comment:
Parameters below default level are used in pr.

If possible, define what it means

Reply:
I referred to definition of log level to define metric level value in this PR:
- 0 disable
- 1 error
- 2 warn
- 3 info
- 4 debug

But I'd like to formalize it later, after receiving more comments.

Reply:
+1. We need to formalize the meaning of the level and have a documentation. Otherwise, one may fail to add new metric with the correct level.

Reply:
We define 4 storage metric levels:
- Disabled = 0,
- Critical = 1,
- Info = 2,
- Debug = 3,

We set several storage metric to `Critical` and `Info`, while leaving others `Debug`.

We expose compute node level `storage_metric_level` in config file, which by default is set to `Info`.